src/containers/HSM/CreateTemplate/CreateTemplate.module.css

```
@@ -0,0 +1,886 @@
+  justify-content: flex-start;
+  gap: 8px;
+}
+
+.MessageBubble {
+  background: #dcf8c6;
+  border-radius: 10px 0 10px 10px;
+  padding: 10px 14px;
+  font-size: 14px;
+  color: #1a1a1a;
+  max-width: 90%;
+  word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)